scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -958,30 +962,16 @@ s32 BPF_STRUCT_OPS(layered_select_cpu, struct task_struct *p, s32 prev_cpu, u64
 	return prev_cpu;
 }
 
-static __always_inline
-bool pick_idle_cpu_and_kick(struct task_struct *p, s32 task_cpu,
-			    struct cpu_ctx *cpuc, struct task_ctx *taskc,
-			    struct layer *layer)
-{
-	s32 cpu;
-
-	cpu = pick_idle_cpu(p, task_cpu, cpuc, taskc, layer, false);
-
-	if (cpu >= 0) {
-		lstat_inc(LSTAT_KICK, layer, cpuc);
-		scx_bpf_kick_cpu(cpu, SCX_KICK_IDLE);
-		return true;
-	} else {
-		return false;
-	}
-}
-
-static __always_inline
-bool try_preempt_cpu(s32 cand, struct task_struct *p, struct cpu_ctx *cpuc,
-		     struct task_ctx *taskc, struct layer *layer,
-		     bool preempt_first)
+/*
+ * XXX - It'd be better to get @cpuc and @enq_flags from the caller but that
```

Comment:
Maybe we need a `struct context_ctx` of all the different contexts or something to make it easier to maintain/verify.  This should be good for now.

Reply:
Yeah, maybe.